release-tools/src/main/java/org/springframework/data/release/jira/JiraIssue.java

```diff
@@ -37,36 +47,168 @@
 	/**
 	 * Returns whether the ticket is only fixed in the given {@link Train}. {@literal false} indicates the {@link Ticket}
 	 * has been resolved for
-	 * 
+	 *
 	 * @param train
 	 * @return
 	 */
 	public boolean wasBackportedFrom(Train train) {
 
 		List<FixVersions> fixVersions = fields.getFixVersions();
-		return !(fixVersions.size() == 1 && fixVersions.get(0).name.contains(train.getName()));
+		return fixVersions != null && !(fixVersions.size() == 1 && fixVersions.get(0).name.contains(train.getName()));
+	}
+
+	/**
+	 * Returns whether the ticket is a release ticket.
+	 *
+	 * @param moduleIteration
+	 * @return
+	 */
+	public boolean isReleaseTicket(ModuleIteration moduleIteration) {
+
+		List<FixVersions> fixVersions = fields.getFixVersions();
+		return fixVersions.size() == 1 && fixVersions.get(0).name.contains(moduleIteration.getTrain().getName())
```

Comment:
That name check of the first element seems to be common (see above). Extract a method?